packages/container/README.md

```diff
@@ -0,0 +1,148 @@
+| width           | string      | Apply CSS width properties               |
+| minWidth        | string      | Apply CSS min-width properties           |
+| maxWidth        | string      | Apply CSS max-width properties           |
+
+
+## Height
+
+| Name            | Type        | Description                              |
+| --------------- | ----------- | ---------------------------------------- |
+| height          | string      | Apply CSS height properties              |
+| minHeight       | string      | Apply CSS min-height properties          |
+| maxHeight       | string      | Apply CSS max-height properties          |
+
+
+## Flex Properties
+
+| Name                | Type        | Description                          |
+| ------------------- | ----------- | ------------------------------------ |
+| flex                | bool        | Apply display: flex                  |
+| inlineFlex          | bool        | Apply display: inline-flex           |
+| flexWrap            | bool        | Apply flex-wrap: wrap                |
+| flexRow             | bool        | Apply flex-direction: row            |
+| flexColumn          | bool        | Apply flex-direction: column         |
+| justifyStart        | bool        | Apply justify-content: flex-start    |
+| justifyEnd          | bool        | Apply justify-content: flex-end      |
+| justifyCenter       | bool        | Apply justify-content: center        |
+| justifySpaceBetween | bool        | Apply justify-content: space-between |
+| justifySpaceAround  | bool        | Apply justify-content: space-around  |
+| alignStart          | bool        | Apply align-items: flex-start        |
+| alignEnd            | bool        | Apply align-items: flex-end          |
+| alignCenter         | bool        | Apply align-items: center            |
+| alignSpaceBetween   | bool        | Apply align-items: space-between     |
+| alignSpaceAround    | bool        | Apply align-items: space-around      |
+| direction           | string      | Apply CSS flex-direction properties  |
+| alignItems          | string      | Apply CSS align-items properties     |
+| alignContent        | string      | Apply CSS align-content properties   |
+| justifyContent      | string      | Apply CSS justify-content properties |
+| alignSelf           | string      | Apply CSS align-self properties      |
+| flexBasis           | string      | Apply CSS flex-basis properties      |
+| flexGrow            | string      | Apply CSS flex-grow properties       |
+| flexShrink          | string      | Apply CSS flex-shrink properties     |
+
+
+## Other
+
+| Name            | Type        | Description                           |
+| --------------- | ----------- | ------------------------------------- |
+| withOutline     | bool        | Add 1px solid red outline             |
+| noPadding       | bool        | Remove default padding from container |
```

Comment:
If I want to find this in the padding section?

Reply:
Great idea. Just pushed this update.